### PR TITLE
publish-Yash's-interview

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR publishes Asynchronous Interview of the MeshMate [Yash Sharma](https://github.com/Yashsharma1911 )

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@Yashsharma1911 for some reasons I'm not able to add you to the reviewers'.
Please have a Review and do suggest changes as you deem fit :)

File: src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+title: "Meet the MeshMate: Yash Sharma"
+subtitle: "An interview series with Layer5 MeshMates"
+date: 2023-05-24 10:30:05 -0530
+author: Vivek Vishal
+thumbnail: "./yash-sharma-layer5-meshmate.png"
+darkthumbnail: "./yash-sharma-layer5-meshmate.png"
+category: Community
+tags:
+ - Community
+featured: true
```
Comment (Chadha93) [inline src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx:11]:
```suggestion

```

**Comment by l5io** [comment]
🚀 Preview for commit eabe72c3b28cb1ddd8f5d8f2a4af7d4f6e0a6a83 at: https://646f7eef571e2010d2f103f1--layer5.netlify.app

**Comment by leecalcote** [comment]
@Yashsharma1911 can now be added as a reviewer.

File: src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx [hunk 2]
```diff
@@ -0,0 +1,192 @@
+---
+title: "Meet the MeshMate: Yash Sharma"
+subtitle: "An interview series with Layer5 MeshMates"
+date: 2023-05-24 10:30:05 -0530
```
Comment (leecalcote) [inline src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx:4]:
```suggestion
date: 2023-05-29 10:30:05 -0530
```

File: src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+author: Vivek Vishal
+thumbnail: "./yash-sharma-layer5-meshmate.png"
+darkthumbnail: "./yash-sharma-layer5-meshmate.png"
+category: Community
+tags:
+ - Community
```
Comment (leecalcote) [inline src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx:10]: Do we have a MeshMate tag already?

**Review by Yashsharma1911 — APPROVED** [review]

**Comment by Yashsharma1911** [comment]
LGTM also thanks for making changes @vishalvivekm and @Chadha93

**Comment by Yashsharma1911** [comment]
@vishalvivekm it's been a week I think we are ready to go here

File: src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx (hunk 3 above)
Comment (vishalvivekm) [reply src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx:10]: Yes we do.

Comment (vishalvivekm) [inline src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx:10]:
```suggestion
 - Community
 - MeshMate
```

File: src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx (hunk 2 above)
Comment (Chadha93) [reply src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx:4]: @vishalvivekm Let's update the date here, and merge.

Comment (vishalvivekm) [reply src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx]: Done @chadha19

**Comment by l5io** [comment]
🚀 Preview for commit 1e1d5113493d01d4af79cf303dd2ef38dc2f9a88 at: https://648197a18c6506042ad98704--layer5.netlify.app

**Comment by vishalvivekm** [comment]
> @vishalvivekm it's been a week I think we are ready to go here

Yup we're now @Yashsharma1911

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**